src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/AccessibilityReporter.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.scalatestaccessibilitylinter
+
+import org.scalatest.events._
+import org.scalatest.{Reporter => ScalaTestReporter}
+import uk.gov.hmrc.scalatestaccessibilitylinter.domain._
+
+import scala.collection.mutable.ListBuffer
+
+@deprecated("This is an experimental api that is subject changes or possible removal.", "0.4.0")
```

Comment:
Used @deprecated to show this API is being incubated / experimental

Reply:
Feels a bit odd adding something that is deprecated from the start. Could we consider leaving out for now until we establish there is a requirement for it?

Reply:
I'm abusing deprecated to give a warning on use that it's experimental just in case someone saw it and ran with it. If we don't want to have any WIP in the main branch that's ok with me, I can extract it 👍